waveform.c

```diff
@@ -308,7 +308,28 @@ static gboolean
 waveform_draw_cb (void *user_data)
 {
     waveform_t *w = user_data;
-    gtk_widget_queue_draw (w->drawarea);
+
+    DB_playItem_t *trk = deadbeef->streamer_get_playing_track ();
```

Comment:
Just when I hit "Merge" I found this issue. There's a leak here, you have to call `deadbeef->pl_item_unref (trk)`after acquiring the track duration.

Reply:
But nvm, it's easier when I'm fixing that.